.github/workflows/added_posts.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           pip3 install -Ur "${GITHUB_WORKSPACE}/notice_communication/requirements.txt"
           readarray -t added_files <<<"$(jq -r '.[]' <<<'${{ steps.files.outputs.added }}')"
           for added_file in ${added_files[@]}; do
-            if [[ $added_file == content/notices/* ]]; then
+            if [[ $added_file == content/temp_disabled_notices/* ]]; then
```

Comment:
Done intentionally so notices don't fire for the category placeholder docs. I will issue a follow-up PR to fix this once this PR has merged.